cli/flags.go

```diff
@@ -110,6 +110,10 @@ var (
 		Usage: "placement strategy to use [" + strings.Join(strategy.List(), ", ") + "]",
 		Value: strategy.List()[0],
 	}
+	flIgnoreInactive = cli.BoolFlag{
+		Name:  " ignoreInactive",
+		Usage: "Do not account for non running containers when scheduling",
```

Comment:
This may need clarification. Swarm always accounts for node port occupation, reserved memory/CPU, label, etc.